documentation/api/io.strimzi.api.kafka.model.EntityTopicOperatorSpec.adoc

```diff
@@ -0,0 +1,69 @@
+Configures the Topic Operator.
+
+[id='property-topic-operator-logging-{context}']
+=== `logging`
+
+The Topic Operator has a configurable logger:
+
+* `rootLogger.level`
+
+The Topic Operator uses the Apache `log4j2` logger implementation.
+
+Use the `logging` property in the `Kafka` resource to configure loggers and logger levels.
```

Comment:
we could highlight that it's needed to be done under the `entityOperator.topicOperator` field of the `Kafka` resource.